src/p2p-file.js

```diff
@@ -0,0 +1,31 @@
+const EventEmitter = require('events')
+const hypercore = require('hypercore')
+const ram = require('random-access-memory')
+
+class P2PFile extends EventEmitter {
```

Comment:
```suggestion
class ChangeLog extends EventEmitter {
```

Why:

* We're not actually dealing with a file but with a higher level of abstraction where we don't care if it is store to a file, to RAM, or whatever 🙂
* Bring domain semantics to our codebase